apps/system/test/unit/app_usage_metrics_test.js

```diff
@@ -693,6 +707,22 @@ suite('AppUsageMetrics:', function() {
       mockSettings.y = '2';
     });
 
+    setup(function() {
+      this.sinon.stub(navigator.mozTelephony, 'dial', function() {
+        return Promise.resolve({
+          result: Promise.resolve({
+            success: true,
+            serviceCode: 'scImei',
+            statusMessage: 'fakeImei'
+          })
+        });
+      });
+    });
```

Comment:
You reuse this mozTelephony.dial stub in a lot of places, maybe put this into a function that you can reuse?